p2p/src/handshake.rs

```diff
@@ -29,13 +29,16 @@ use crate::peer::Peer;
 use crate::types::{Capabilities, Direction, Error, P2PConfig, PeerInfo, PeerLiveInfo};
 
 const NONCES_CAP: usize = 100;
+const ADDRS_CAP: usize = 10;
```

Comment:
It would be nice to comment on why these magic numbers are chosen.

Reply:
agree~